framework/src/media/audio/audio_manager.h

```diff
@@ -376,6 +436,149 @@ audio_manager_result_t set_input_audio_gain(uint8_t gain);
  ****************************************************************************/
 audio_manager_result_t set_output_audio_volume(uint8_t volume);
 
+/****************************************************************************
+ * Name: foreach_supported_process_device
+ *
+ * Description:
+ *   Find stream in device that is supports specific process on dsp
```

Comment:
that is supported by dsp

Reply:
on is correct, usually using 'voce activty on the bsp' or 'voice detector on dps h/w'